docs/features.md

```diff
@@ -2,4 +2,35 @@ Click the below for a walk through of the product functionality:
 
 [![PostHog demo](_media/posthog-demo.png)](https://youtu.be/vGy9gzMAAes-Y?t=35s "PostHog Demo Video")
 
-There is a detailed table of the functionality on the [pricing](/pricing) page.
\ No newline at end of file
+There is a detailed table of the functionality on the [pricing](/pricing) page.
+
+Below is a table of PostHog features and how this compares with the functionality in common alternatives.
+
+
+|  Feature 	|   PostHog	| Amplitude  	|  Mixpanel 	| Heap  	|
+|:-:	|:-:	|:-:	|:-:	|:-:	|
+|   Autocapture	|   ✔	|   	|   	|   ✔	|
+|   API 	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   In-App Events Tracking	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   Conversion Tracking	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   Dashboard	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   Data Visualisation	|  ✔ 	|   ✔	|   ✔	|   	|
+|   Cohort Analysis	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   Funnel Analysis	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   Push Notifications	|   	|   ✔	|   ✔	|   ✔	|
+|   Retention Tracking	|   	|   ✔	|   ✔	|   ✔	|
+|   Revenue Tracking	|   	|   ✔	|   ✔	|   ✔	|
+|   Uninstall Tracking	|   	|   ✔	|   	|   	|
+|   Data Import	|   	|   ✔	|   ✔	|   	|
+|   Data Export	|   	|   ✔	|   ✔	|   	|
+|   External Integrations	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   Keyword Tracking	|   	|   ✔	|   ✔	|   ✔	|
+|   Link Tracking	|   	|   	|   ✔	|   	|
```

Comment:
I think this is specifically "email link tracking?"

Reply:
Yes can change